modules/pages/client/components/Home.component.js

```diff
@@ -79,14 +80,26 @@ export function getSignupUrl(circleSlug) {
   return '/signup';
 }
 
+function rounded(number, round = 1000) {
+  return parseInt(number / round) * round;
+}
+
 export default function Home({ user, isNativeMobileApp, photoCredits }) {
   const { t } = useTranslation('pages');
   // `tribe` route supported for legacy reasons, deprecated Feb 2021
   const { circle: circleRouteParam, tribe: tribeRouteParam } = getRouteParams();
   const circleRoute = circleRouteParam || tribeRouteParam;
 
+  const [statistics, setStats] = useState(false);
+  useEffect(async () => {
+    const { data } = await statisticsAPI.get();
```

Comment:
This API returns a whole lot of things, and does some ~10-15 database queries to get all that data:

https://github.com/Trustroots/trustroots/blob/0ef09face509894c22bbd79f98ea9b9fbe1a5eda/modules/statistics/server/controllers/statistics.server.controller.js#L244-L247

It's driving the public stats page https://www.trustroots.org/statistics

We can probably handle that for every logged out homepage load, but on the other hand it would be small effort to add new API route returning only user count. :-) Would you be up for it?

If yes, the route config is here: https://github.com/Trustroots/trustroots/blob/master/modules/statistics/server/routes/statistics.server.routes.js

It ties to a function that returns the content in controller: https://github.com/Trustroots/trustroots/blob/master/modules/statistics/server/controllers/statistics.server.controller.js

and there's a helper to just get the user count: 
https://github.com/Trustroots/trustroots/blob/0ef09face509894c22bbd79f98ea9b9fbe1a5eda/modules/statistics/server/controllers/statistics.server.controller.js#L22

You could add the rounding directly in the API if you'd like.

Reply:
hey Mikael, good feedback, thanks! Will get back to it whenever I have a moment. 

I am actually really very glad you did the review, thanks!

Reply:
no problem, thanks for patience! :-)

Reply:
I didn't have time to look at this this week and I am travelling next 2-3 weeks, so in the meantime opened #2394. 

From Sept I hope to have a bit more time 🤞🏼 for everything other than my kiddo, as the person who takes care of her will be back and we are moving forward with our child care/forest kindergarten plans. (child care is unimaginably neglected in Poland, mostly relies on grandparents who are not an option for us :/)